lib/styles/components/_modal.scss

```diff
@@ -15,7 +15,7 @@
     width: 50%;
     max-width: 100%;
     max-height: 100vh;
-    overflow-y: auto;
+    overflow-y: visible;
```

Comment:
I had to change this to `visible` because on smaller modals, the datepicker was causing scroll:

![Modal Overflow-high](https://user-images.githubusercontent.com/22348659/144406480-bff86d75-44fe-455e-b6ce-342903ca5acd.gif)